### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by cyouh95** [tinman777/intro-data-capstone-biodiversity#5]
(Exceeds Expectations)

Well done on this project! Good job on all the calculations as well as data visualizations. In addition, great that you are not just showing the results of your calculations but also interpreting what it signifies! One suggestion though would be to summarize your talking points into perhaps bullet points instead of writing out all of what you would say in the slides.

As a further challenge, continue exploring these datasets and see if you could ask some more research questions about it. Try designing your own tests and continue to play around with different ways of data visualizations. Have fun!

**Comment by tinman777** [comment]
I have a question does this mean I am finished with the program and if so, how will I be notified?

**Comment by tinman777** [comment]
Is there a certificate that is earned at the end of the program?

**Comment by cyouh95** [comment]
Yes, congratulations on finishing the course! You will be receiving a digital certificate of completion via email.
